.github/workflows/presubmit-compositions.yaml

```diff
@@ -23,7 +23,48 @@ on:
     branches: ["master"]
     paths:
       - "experiments/compositions/**"
+
+env:
+  GOWORK: off
 jobs:
+  verify-goimports:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          stable: true
+      - uses: actions/checkout@v4
+      - name: verify goimports
+        run: |
+          files=$(go run golang.org/x/tools/cmd/goimports -format-only -l experiments/compositions/.)
```

Comment:
It may be easier to reproduce any issues w this cmd if we throw these into a script file